.github/workflows/markdown-link-check-pr.yaml

```diff
@@ -0,0 +1,38 @@
+name: PR check Markdown links
+
+# Only run this workflow on PRs
+on:
+  pull_request:
+    types: [opened, synchronize, reopened, ready_for_review]
+
+# Disable permissions for all available scopes for the GITHUB_TOKEN
+# except for the metadata scope.
+permissions: {}
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  pr-markdown-link-check:
+    # Only run the job on non-draft PRs
+    if: "github.event.pull_request.draft == false"
+    name: Check PR Markdown links
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out code under the $GITHUB_WORKSPACE directory
+        uses: actions/checkout@v3
+      - name: Check markdown links
+        uses: gaurav-nelson/github-action-markdown-link-check@1.0.15
+        with:
+          use-quiet-mode: 'no'
+          use-verbose-mode: 'yes'
+          config-file: './.markdown-link-check/markdown-link-check-config.json'
+          check-modified-files-only: 'no'
```

Comment:
This is intentional:

All the specified md files and directories in the workflows are
intentionally checked, independently on whether they have been
modified or not. The reason for this is that even if a given .md
file has not been modified, its links can become broken. For example,
if a PR moves .md files from directories and some other .md files
contain relative links referencing the old path of the moved files,
then the links become broken even though the file containing the
references not being modified.